coco/request_forwarder.py

```diff
@@ -54,6 +56,40 @@ def add_endpoint(self, name, endpoint):
         """
         self._endpoints[name] = endpoint
 
+    # We need a separate server to track metrics produced by this process.
+    # This should be called by the worker process when it is started.
+    @staticmethod
+    def start_prometheus_server(port=9090):
```

Comment:
Let's define the default value when loaded from config. Then it also makes more sense to write it down as a default value for the config documentation (once that exists...).